### ISSUE TITLE
Bug - Cannot send with too many UTXO

### ISSUE THREAD

**Issue by SpicyPete** [Bitcoin-com/badger-mobile#77]
# Issue

If a wallet has too many UTXO's it is signing with all of them, causing errors during sending

## Solution

Only gather enough UTXO's to use within the transaction, do not include all.

**Comment by SpendBCH** [comment]
Reposting from #78 

We may also need to account for similar UTXO issues with SLP transactions and how to handle Send Max when more than one transaction is required

**Comment by SpicyPete** [comment]
Good catch, will update issue and leave it open until that case is addressed as well

**Comment by SpicyPete** [comment]
Update - Issue was also for SLP transactions.  Fixed in recent PR.  
Send Max is only outstanding issue with this currently.

**Comment by mspaansen** [comment]
Unknown
